bigtop.mk

```diff
@@ -288,3 +288,16 @@ AVRO_SITE=$(APACHE_MIRROR)$(AVRO_DOWNLOAD_PATH)
 AVRO_ARCHIVE=$(APACHE_ARCHIVE)$(AVRO_DOWNLOAD_PATH)
 $(eval $(call PACKAGE,avro,AVRO))
 
+# Impala
+IMPALA_NAME=impala
+IMPALA_RELNOTES_NAME=Impala
+IMPALA_PKG_NAME=impala
+IMPALA_BASE_VERSION=2.2.0
+IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.3.0
+IMPALA_RELEASE_VERSION=1
+IMPALA_TARBALL_DST=impala-$(IMPALA_BASE_VERSION).tar.gz
+IMPALA_TARBALL_SRC=impala-$(IMPALA_BASE_VERSION)-cdh5.4.5-rocana1.3.0.tar.gz
```

Comment:
I noticed this path has `cdh5.4.5` in it but IMPALA_PKG_VERSION has `hdp2.3.0` in it.